### PR TITLE
test/utils: fix some issues

### PR DESCRIPTION
**Pull request by zhouhao3**
**What this PR does / why we need it**:
Fix some issues in the tests/utils.go file.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
NONE

**Special notes for your reviewer**:

**Release note**:

```release-note

NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -914,9 +914,9 @@ func EnsureKVMPresent() {
 				virtHandlerNode, err := virtClient.CoreV1().Nodes().Get(pod.Spec.NodeName, metav1.GetOptions{})
 				ExpectWithOffset(1, err).ToNot(HaveOccurred())
 
-				kvmAllocatable, ok := virtHandlerNode.Status.Allocatable[services.KvmDevice]
-				vhostNetAllocatable, ok := virtHandlerNode.Status.Allocatable[services.VhostNetDevice]
-				ready = ready && ok
+				kvmAllocatable, ok1 := virtHandlerNode.Status.Allocatable[services.KvmDevice]
+				vhostNetAllocatable, ok2 := virtHandlerNode.Status.Allocatable[services.VhostNetDevice]
+				ready = ready && ok1 && ok2
```
Comment (davidvossel) [inline tests/utils.go:919]: ha, This is great. so before this we could technically start scheduling VMs before the kvm resource is ready.

File: tests/utils.go [hunk 2]
```diff
@@ -2949,6 +2953,7 @@ func WaitUntilVMIReadyWithContext(ctx context.Context, vmi *v1.VirtualMachineIns
 
 	// Fetch the new VirtualMachineInstance with updated status
 	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
```
Comment (davidvossel) [inline tests/utils.go:2956]: instead of panic here. I think `Expect(err).ToNot(HaveOccurred())` is more appropriate

**Review by davidvossel — COMMENTED** [review]
/ok-to-test

File: tests/utils.go (hunk 2 above)
Comment (zhouhao3) [reply tests/utils.go]: updated, PTAL.

**Comment by zhouhao3** [comment]
@davidvossel @rmohr PTAL

**Pull request merged**